projects/fal_client/pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "fal_client"
```

Comment:
this is kind of the thing we were joking about with having over-specified package and module names. is the idea for this to be an entirely separate pypi package installed as `fal-client` or something installable as `fal[client]`?

Reply:
I think since we are in a hurry `fal-client` for now and eventually a nice thing like `fal` for this and `fal[sdk]` for the full thing

Reply:
Shouldn't this name field be `fal-client` rather than `fal_client`?

Reply:
@squat fal-client is the same fal_client (they are normalized). https://packaging.python.org/en/latest/specifications/name-normalization/